dispatch/api/views.py

```diff
@@ -93,6 +99,28 @@ def get_queryset(self):
 
         return queryset
 
+    @detail_route(permission_classes=[AllowAny], methods=['post'],)
+    def suggested(self, request, parent_id=None):
+        article = get_object_or_404(Article.objects.all(), pk=parent_id)
+
+        # Update count
+        if 'article_id' in request.data:
+            article_id = request.data['article_id']
+            if article_id == parent_id:
+                return Response('')
+
+            related_article = get_object_or_404(Article.objects.all(), pk=article_id)
+
+            relation, created = ArticleRelation.objects.get_or_create(parent=article, article=related_article)
+
+            if not created:
+                relation.count += 1
+                relation.save()
+        else:
+            return Response('')
+
+        return Response('')
```

Comment:
Is this method supposed to be part of this PR?

Reply:
removed, along with other suggested article stuff